src/login/resources/themes/purple/theme.json

```diff
@@ -4,7 +4,7 @@
     "authorUsername": "purple",
     "authorName": "Steve Greene",
     "stylesheets": [
-        "../dark/dark.css"
```

Comment:
You need to inherit the dark theme's CSS in order to get the white logo/settings gear. The default colors are black (i.e. the `light` theme).

Reply:
Just leave this line and add a comma, you can have multiple CSS files.

Reply:
I wanted to try out the theme with the black logo/gear instead of the white.  Anything else need to change for that to happen?